### PR TITLE
Migration to typescript

### PR DESCRIPTION
**Pull request by frankyhollywood**

### CODE REVIEW

**Comment by ewelinajozwik** [comment]
Build is failing due to linting errors

File: projects/mercury/.eslintrc.json
Comment (ewelinajozwik) [inline projects/mercury/.eslintrc.json:1]: why is this one removed?

File: projects/mercury/README.md [hunk 1]
```diff
@@ -1,87 +1,46 @@
-# Mercury
+# Getting Started with Create React App
```
Comment (ewelinajozwik) [inline projects/mercury/README.md:1]: You overwrited the original documentation, could you revert this change?

**Review by ewelinajozwik — CHANGES_REQUESTED** [review]
Why did you changed some .js files to .tsx?

Also, you applied some weird formatting to almost all the changed files. Because of that most of them are not recognized here as renaming change, but as deletion of one file and adding a new one with .ts/.tsx extension.
Look at the difference in length between old and new files.

File: projects/mercury/.eslintrc.json
Comment (frankyhollywood) [reply projects/mercury/.eslintrc.json:1]: I have added it again

File: projects/mercury/README.md (hunk 1 above)
Comment (frankyhollywood) [reply projects/mercury/README.md:1]: yes :)
